site/templates/features.php

```diff
@@ -9,7 +9,7 @@
       <section class="features-section">
         <?= snippet('features/header', [
           'heading' => 'The Panel',
-          'subheading' => 'A control room that feels like home'
+          'subheading' => 'A control room where you will instantly feel at home'
```

Comment:
Personally, here I like Bastian's original line better. Kurz und knack :D

Reply:
Maybe we can find something better, but feels like home sounds strange to me in combination with a control room.

Reply:
Maybe we should remove the "control room" part entirely and make it more human friendly.